backup.js

```diff
@@ -55,7 +55,7 @@ function bangleDownload() {
     showToast('Backup complete!', 'success');
     if (typeof Android !== "undefined" && typeof Android.saveFile === 'function') {
       // Recent Gadgetbridge version that provides the saveFile interface
-      Android.saveFile("Banglejs backup.zip", "application/zip", content);
+      Android.saveFile("Banglejs backup.zip", "application/zip", btoa(content));
```

Comment:
Does this fix a bug? Let's pull it out to a separate PR if so

Reply:
I have no clue what this is? I didn't create a backup.js

Reply:
No problem - looks like it's a github thing, seems fine locally